src/components/node-list/node-list-items.js

```diff
@@ -176,14 +223,16 @@ export const getFilteredNodeItems = createSelector(
 
 /**
  * Get formatted list of sections
+ * @param {boolean} flag value of modularpipeline flag
  * @return {array} List of sections
  */
-export const getSections = createSelector(() =>
-  Object.keys(sidebar).map((name) => ({
+export const getSections = (flag) => {
+  const sidebarObject = sidebar(flag);
```

Comment:
I think `sidebarObject` is probably the `sidebarConfig` here?

Reply:
sure however it's just a difference of preference in naming?

Reply:
Yeah in this case the name can be more specific since we know it's a config object, right?